### DISCUSSION TITLE
Complex Adaptive System Resilience vs Antifragile

### DISCUSSION THREAD

**Discussion by JRBliekendaal** [JRBliekendaal/master-thesis#13] [General]
@edzob 

It was a very nice webinar last Friday and I only asked the question on research because nobody was asking questions. This was just to show people that there is not that much research done on this subject yet. Your findings are unique at this moment.

The main part of your webinar was about CAS Resilience. And you used an example that is also used by Taleb (2013, p. 39) on becoming more antifragile. That example used the Mithridate method. Just give a little poison every day so that people get used to it (in changing rapidly). Also, you talked about the variability (which is also often used with Agility frameworks like SAFe and antifragile (Taleb, 2013).

It seems to me (based on your story) that CAS Resilience is already very close to Antifragility (when I believe the diagrams you used in the webinar but also in your thesis). CAS increases the value over time when you are under stress.

The main difference between them only that with CAS you first lose value before you are gaining value (the little dip in the start) instead of gaining value (from the start) with antifragility? In the end, both are gaining and not losing.

But what are the key differences between CAS Resilience and Antifragile besides the pattern over time? Can you explain to me very short what the key difference is between CAS Resilience and Antifragility?

**Comment by JRBliekendaal** [comment]
And yes I have read your thesis and did see the EAAL model ;) but still, it is a very thin line between both.

**Comment by edzob** [comment]
What I found in the literature:
The difference is the labels on the axis.
(CAS) resilience is a function over time with value as output..
(anti) fragile is a function over stress with value as output.

I think it will be possible to state that antifragile is the differential equitian of cas resilience
I think this is the reason why the atributes in the literature that talk about implementing AF are different then what taleb writes..

**Reply by edzob** [reply]
The final comment shoudl be a bit nuanced. Since I put opationalit from Taleb into diversity.. 
But in all the points Taleb makes about AF are ways to optimize CAS Resilience..
IMHO i think this is to expensive and disrubtive for most companies/organisations. And for example the seneca barbell is difficult to implement in all dimensions.. 

I think that the trick is that controlled design is possible for Engineering Resilience and Systems Resilience

And that the amplification of variety states that you need to go with the flow.
That is the difficult part that charles addressed int he webinar.. 
decoupling and networking..  that leads to more chaotic behavour.. and that is inevetible (like tanos)
but by embracing chaos and learning how to exploit it.. you can grow stronger of the extra variety you create in the system and win (like Iron Man). :)

**Reply by JRBliekendaal** [reply]
I will put on one of my Marvel T-Shirts from my Extensive collection next time we meet.

Thanks. What I read is that it is really in the nuance and more people struggle with it. I see it differently as you do. CAS Resilience is still more of a Phoenix with ambitions but the Phoenix will never become a Hydra.

**Reply by edzob** [reply]
Nice.. I like people that see thinks differently.

Do you also see a phoenix analogy as somehting that dies and comes back as the same?
And hydra as something that dies a little bit and becomes stronger (head killed is different from body slain).

I dont see CAS resilence vs AF as a nuance..
I think CAS Resileince is to do for an organisation (teams) but AF is almost only for small organisations or self-employed, since designign this for a whole organisation demands a lot of smart governance. 
But there is much to be researched on this and learned by me. Looking forward talking about this, or reading your reflections on this..

**Comment by JRBliekendaal** [comment]
My basic attitude is that of a mathematician./scientist. Sometimes I am even very binary and very sceptical when something is somewhere in between ;) I really do not like shades of grey. So clear definitions is what I pursue. For example, I often think about what infinitive is. I do not believe in infinity Everything has an end, a number, a size, a... Sometimes I use philosophy to explaining things (made a model once to place DEMO in the right perspective based on the theory of essentialism :D) Everything needs to be explained. When I cannot explain it I do not fall back to religion (I am an atheist) or infinity or approximately. I only accept that we did not found out the answer yet.

I indeed see a phoenix as something that dies and has a rebirth but exactly the same as it was. The Hydra gains a lot when you chop of a head. But the Hyrdra also dies in the end. It is not infinitive.

My first research subject for the AMS was btw how to make an enterprise organisation withstand disruptions by splitting the enterprise up into a system of startups. In this way, you have smaller pieces (fragile) but the system itself will be antifragile. Taleb has some really good examples of this. Yes, I Agree the current organisational configurations cannot get antifragile. it is not in their nature. They are about governance and get and stay in control of things. When you want to be antifragile you have to have decentralised decision-making and self-determination. Almost Anarchistic. As a company, you have to embrace chaos to survive in chaos. An ecosystem of startups fits better in this direction.

So what about CAS? As I see it at this moment is as follows. It's a Phoenix+ and still not hydra. You probably can say that the Phoenix that pursues CAS will rebirth with memory on what not to do anymore (learning capability). In this way, it can arm itself and create more value because it gets more resilient by becoming more agile. (damn another discussion. the difference between Robust, Agile and Resilience). With my definitions, I even think that Agility is higher on the ladder than Resilience ;) See my one-pager document for the definitions I use for this.

my opinions no academic grounding yet

**Reply by JRBliekendaal** [reply]
Btw one of my pitfalls with research is that I stay too long with a concept because when I do not understand it fully I cannot proceed ;) When I understand something I do not have any issues anymore with putting it down on paper.

**Reply by edzob** [reply]
Interresting.. I think many scientist are of this school you describe.

1. Did you read black swan (by taleb) or the first thirth of Goedel Escher Bach (goedels incompleteness theorem) ?
2. Did I send you my draft whitepaper on Design for Chaos?

**Reply by JRBliekendaal** [reply]
You are referring to the part of Achilles, the Turtoise and the crab? Long time since I read it. I only remember that my head was spinning at the end. 🤣 and yes I did read the latter first but a very long time ago. I found the explanation of how the music of Bach worked very inspiring. It is only one big pattern. Hofstadter is really a erudite with a lot of different points of view. Slightly autistic 🤣 but I loved it after I finished it. Very hard to read. I will read it again somewhere in time when I find the paperback I have. Could be that it will be a bit out dated. I guess some philosophic themes he used are already superseded by new ones. 

And no you did not send me your paper **yet** 😉

**Reply by JRBliekendaal** [reply]
This book is also in line to be read. But I think it will not be before May 2022. 

The Beginning of Infinity: Explanations That Transform the World By David Deutsh

**Comment by edzob** [comment]
the part of GED i tried to refer to is in the Wikipedia link.

this is comparable to the black swan (taleb), what we can learn from
spaghetti saus (gladwell) and the Parabel of the cave (Plato)

Goedel wrote the story in the language of logic. where the others from that
list used different languages.

i think you might find this helpful

https://en.m.wikipedia.org/wiki/G%C3%B6del%27s_incompleteness_theorems

The first incompleteness theorem states that no consistent system of axioms
whose theorems can be listed by an effective procedure (i.e., an algorithm)
is capable of proving all truths about the arithmetic of natural numbers.
For any such consistent formal system, there will always be statements
about natural numbers that are true, but that are unprovable within the
system. The second incompleteness theorem, an extension of the first, shows
that the system cannot demonstrate its own consistency.

Edzo

-- 
"It is amazing what you can accomplish if you do not care who gets the
credit." - Harry S Truman

Mobile, Signal, WhatsApp -
+31629485425
Twitter, riot.im, wire, Matrix -
@edzob

Homepage - https://www.edzob.com/page/about
ORCID - https://orcid.org/0000-0003-0097-7375
